src/style.css

```diff
@@ -2,0 +1,30 @@
+.body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Using `box-sizing: border-box;` for all elements can simplify your layout and make it more predictable. You can do this using a universal selector `*` and its pseudo-elements. Here's how you can do it: css
*, *::before, *::after {
  box-sizing: border-box;
}
```